### PR TITLE
Enhance skuba addon upgrade plan

### PR DESCRIPTION
**Pull request by jenting**
# Describe your changes

The `skuba addon upgrade plan` also do Kubernetes server-side dry-run validation.

# Related Issues / Projects

Fixes: https://github.com/SUSE/avant-garde/issues/1765

### CODE REVIEW

File: adoc/admin-updates.adoc [hunk 1]
```diff
@@ -172,7 +173,7 @@ That warning means that Cilium is going to regenerate all internal data on the f
 It can result in temporary connection loss for pods and services which might take few minutes.
 ====
 
-In order to get an overview of the addon updates available, you can run:
+To get an overview of the addon updates available and validates the addon base and patches manifest before applied to the current cluster, you can run:
```
Comment (innobead) [inline adoc/admin-updates.adoc:176]:
```suggestion
To get an overview of the addon updates available with validating the addon base and patches manifest before applied to the current cluster, you can run:
```

**Comment by jenting** [comment]
Some problems in Travis CI :unamused: 
```
$ wget https://raw.githubusercontent.com/openSUSE/doc-ci/master/travis/travis.sh
--2020-07-13 05:29:42--  https://raw.githubusercontent.com/openSUSE/doc-ci/master/travis/travis.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133, 151.101.64.133, 151.101.128.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.
HTTP request sent, awaiting response... 500 Internal Server Error
2020-07-13 05:29:42 ERROR 500: Internal Server Error.
The command "wget https://raw.githubusercontent.com/openSUSE/doc-ci/master/travis/travis.sh" failed and exited with 8 during .
```

**Comment by jenting** [comment]
Travis CI is happy now :smile:

**Comment by jenting** [comment]
skuba PR merged, It's ready to review~

**Comment by jordimassaguerpla** [comment]
Not in 5.0.0

**Pull request merged**